embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/crumbs/EmbraceBreadcrumbService.kt

```diff
@@ -44,22 +32,15 @@ internal class EmbraceBreadcrumbService(
     private val logger: InternalEmbraceLogger = InternalStaticEmbraceLogger.logger
 ) : BreadcrumbService, ActivityLifecycleListener, MemoryCleanerListener {
 
-    /**
-     * A deque of breadcrumbs.
-     */
-    private val viewBreadcrumbs = LinkedBlockingDeque<ViewBreadcrumb?>()
     private val customBreadcrumbDataSource = CustomBreadcrumbDataSource(configService)
     private val webViewBreadcrumbDataSource = WebViewBreadcrumbDataSource(configService)
     private val rnBreadcrumbDataSource = RnBreadcrumbDataSource(configService)
     private val tapBreadcrumbDataSource = TapBreadcrumbDataSource(configService)
     private val viewBreadcrumbDataSource = ViewBreadcrumbDataSource(configService, clock)
+    private val fragmentBreadcrumbDataSource = FragmentBreadcrumbDataSource(configService, clock)
```

Comment:
in this going to be injected in the future?

Reply:
Yep, in future this will be declared in a module.